soroban-spec/src/gen/typescript/project_template/src/index.ts

```diff
@@ -0,0 +1,87 @@
+import * as SorobanClient from 'soroban-client';
+import { xdr } from 'soroban-client';
+import { Buffer } from "buffer";
+import { scValStrToJs } from './convert';
+import { invoke, InvokeArgs } from './invoke';
+
+export * from './constants'
+export * from './server'
+export * from './invoke'
+
+export type u32 = number;
+export type i32 = number;
+export type u64 = bigint;
+export type i64 = bigint;
+export type u128 = bigint;
+export type i128 = bigint;
+export type u256 = bigint;
+export type i256 = bigint;
+export type Address = string;
+export type Option<T> = T | undefined;
+
+/// Error interface containing the error message
+export interface Error_ { message: string };
+
+export interface Result<T, E = Error_> {
+    unwrap(): T,
+    unrap_err(): E,
+    isOk(): boolean,
+    isErr(): boolean,
+};
+
+export class Ok<T> implements Result<T> {
+    constructor(readonly value: T) { }
+    unrap_err(): Error_ {
```

Comment:
Camel casing? and typo
```suggestion
    unwrapErr(): Error_ {
```